crates/ruff_python_ast/src/visitor.rs

```diff
@@ -739,7 +739,7 @@ pub fn walk_pattern_keyword<'a, V: Visitor<'a> + ?Sized>(
 }
 
 pub fn walk_f_string<'a, V: Visitor<'a> + ?Sized>(visitor: &mut V, f_string: &'a FString) {
-    for f_string_element in &f_string.elements {
+    for f_string_element in f_string.elements.iter() {
```

Comment:
Did you consider adding an `IntoIterator` implementation for `&'a FStringElements`? I think you could avoid changes like this if you did.

```rs
impl<'a> IntoIterator for &'a FStringElements {
    type IntoIter = Iter<'a, FStringElement>;
    type Item = &'a FStringElement;
    fn into_iter(self) -> Self::IntoIter {
        self.iter()
    }
}

Reply:
No I didn't consider that although it makes sense to implement it. Thanks!

Reply:
Are you going to do this in a followup?

Reply:
I've implemented it.

Reply:
Oh my, how did I lost that change lol. I'll implement it in a follow-up then